plugins/hls-class-plugin/test/Main.hs

```diff
@@ -14,18 +14,22 @@ import qualified Ide.Plugin.Class        as Class
 import qualified Language.LSP.Types.Lens as J
 import           System.FilePath
 import           Test.Hls
+import Development.IDE.Types.Logger
+import Data.Functor.Contravariant (contramap)
```

Comment:
Could you format this? (with pre-commit hook or cmd line stylish-haskell)

Reply:
Done.